relays/generic/utils/src/metrics/global.rs

```diff
@@ -0,0 +1,109 @@
+// Copyright 2019-2021 Parity Technologies (UK) Ltd.
```

Comment:
There's now single file in this folder, but I have another one to add in the next PRs